packages/SwingSet/test/test-controller.js

```diff
@@ -258,6 +279,10 @@ test.serial('bootstrap export', async t => {
   }
 
   t.deepEqual(c.dump().log, []);
+  for (let i = 0; i < 7; i += 1) {
+    // eslint-disable-next-line no-await-in-loop
+    await stepGC(); // vat starts
```

Comment:
but not this

huh, with only 6 non-`setup` vats, I'd expect the loop should need 6 iterations, not 7. And the deleted `stepGC` below (the `// dropExports`) one sort of correlates with that. But I'm guessing the test failed when you do 6 loops and retain the `// dropExports` one?

Reply:
I'm reopening this as I was just stung by this in my change to the run queue. Switching to `6` and restoring the second `stepGC` below fixed the issue (after a while of tracing the diff).